src/handlers/shared/check-assignments.ts

```diff
@@ -0,0 +1,65 @@
+import { Context, IssueEvent } from "../../types";
+
+export async function checkPreviousAssignments(context: Context, sender: Context["payload"]["sender"]): Promise<boolean> {
```

Comment:
since this function returns a boolean, the name is a bit weird. maybe something like `hasUserBeenUnassigned` or something similar